test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpClientTests.cs

```diff
@@ -26,7 +26,7 @@ public void HttpClient()
         {
             int expectedSpanCount = EnvironmentHelper.IsCoreClr() ? 2 : 1;
```

Comment:
With the very exhaustive new samples that call every public HttpClient method, I was expecting to get 1 span per HttpClient method call. Could you also add that instrumentation in this PR or maybe it can come in a follow-up PR?

Reply:
I can try adding additional instrumentation methods. I recall that many of them go through SendAsync, but I need to revisit that.